data/helpers.d/network

```diff
@@ -47,7 +47,7 @@ ynh_find_port () {
 # | arg: path - the web path to check the availability of
 ynh_webpath_available () {
 	local domain=$1
-	local path=$(ynh_normalize_url_path $2)
+	local path=$2
```

Comment:
Because, if it seems a good idea to correct the path in `ynh_webpath_register` (because we gonna reserve this path).
I think when you simply check the path, you should use the real path like it's stored or defined.

And, in my commit, `ynh_webpath_available` was called just after a call to `ynh_normalize_url_path`

But, what do you mean by "ynh_webpath_register already checks that the url is available"?
`register-url` uses `url-available` before?

In this case, my commit is useless.